.github/renovate.json

```diff
@@ -18,6 +18,11 @@
       "matchManagers": ["github-actions"],
       "groupName": "GitHub Actions"
     },
+    {
+      "matchDepTypes": ["devDependencies"],
+      "matchUpdateTypes": ["patch", "minor"],
+      "groupName": ["DevDependencies (non-major)"]
```

Comment:
can we add an automerge: true for these?

Reply:
yeah, I thought we wanted a "wait and see" approach